### PR TITLE
docs(presto): Melhorias e Overview e descrevendo lifecycle do Order

### PR DESCRIPTION
**Pull request by gabriel-laet**

### CODE REVIEW

File: src/docs/presto/order-lifecycle/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Order Lifecycle
+
+* After creating [an Order](/presto/create-retail-order), we’ll try to find a nearby Driver to perform the delivery. The current status at this moment is `allocating`.
+* When a Driver is found, the status is changed to `accpeted`.
```
Comment (Piercio-loggi) [inline src/docs/presto/order-lifecycle/index.md]: s/is chagend to/changes to/

File: src/docs/presto/order-lifecycle/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* Once the driver arrives at your shop, the status is updated to `started`.
+* When all packages are delivered to the customers, the status is changed to `finished`.
+* If we can’t find a Driver within 10 minutes, your Order will change status to `dropped`. We’ll not automatically retry, unless you explicit call our [Order Redo endpoint](/presto/order-redo).
+* When an Order is [cancelled](/corp-overview/cancel-order):
+  * Before a Driver is assigned, status will be `cancelled`, meaning you won't be charged.
```
Comment (Piercio-loggi) [inline src/docs/presto/order-lifecycle/index.md]: If no driver has been assigned yet

File: src/docs/presto/order-lifecycle/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  * After a Driver is assigned, status will be `cancelledWithCharge`, meaning we will charge you the minimun value for a delivery.
```
Comment (Piercio-loggi) [inline src/docs/presto/order-lifecycle/index.md]: If a driver was already assigned

File: src/docs/presto/overview/index.md [hunk 4]
```diff
@@ -15,3 +15,18 @@ With the ```shopId``` in hands, you can follow our recommended work-flow:
 - Create an order, through the mutation [createRetailOrder](/presto/create-retail-order)
 
 - Verify the status of your delivery, through the query [packageStatus](/presto/package-status)
+
+### Basic Concepts
+* Shop: indicates the pickup location
+* Package: indicates the delivery location
+* Order: a collection of packages - a single Driver is assigned  to perform the delivery
```
Comment (Piercio-loggi) [inline src/docs/presto/overview/index.md]: tem 2 espaços entre `assigned` e `to`

File: src/docs/presto/overview/index.md [hunk 5]
```diff
@@ -15,3 +15,18 @@ With the ```shopId``` in hands, you can follow our recommended work-flow:
...
+
+#### Shop
+* You can’t create a Shop through our API, this is done through our support.
+* You can list all shops assigned to your account. You’ll need this endpoint to find the Shop ID, which is required when creating an Order.
+
+#### Package
+* A package is composed by three groups: Recipient, Destination & Charge Information
+* *Recipient:* name (required) & phone (once provided, we’ll send an SMS with delivery tracking)
```
Comment (Piercio-loggi) [inline src/docs/presto/overview/index.md]: if provided

File: src/docs/presto/overview/index.md [hunk 6]
```diff
@@ -15,3 +15,18 @@ With the ```shopId``` in hands, you can follow our recommended work-flow:
...
+* *Destination:* delivery address info (lat/lng, street address, street number, neighborhood and city)
+* *Charge Information:* charge method, charge value and charge change (when cash is the chosen charge method)
```
Comment (Piercio-loggi) [inline src/docs/presto/overview/index.md]: when the chosen charge method is cash

File: src/docs/presto/order-lifecycle/index.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# Order Lifecycle
+
+* After creating [an Order](/presto/create-retail-order), we’ll try to find a nearby Driver to perform the delivery. The current status at this moment is `allocating`.
+* When a Driver is found, the status changes to `accpeted`.
```
Comment (gabisurita) [inline src/docs/presto/order-lifecycle/index.md]: typo: accepted

File: src/docs/presto/order-lifecycle/index.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# Order Lifecycle
+
+* After creating [an Order](/presto/create-retail-order), we’ll try to find a nearby Driver to perform the delivery. The current status at this moment is `allocating`.
```
Comment (gabisurita) [inline src/docs/presto/order-lifecycle/index.md]: nitpick `[creating an Order]`

File: src/docs/presto/order-lifecycle/index.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+* When a Driver is found, the status changes to `accpeted`.
+* Once the driver arrives at your shop, the status is updated to `started`.
+* When all packages are delivered to the customers, the status changes to `finished`.
+* If we can’t find a Driver within 10 minutes, your Order will change status to `dropped`. We’ll not automatically retry, unless you explicit call our [Order Redo endpoint](/presto/order-redo).
+* When an Order is [cancelled](/corp-overview/cancel-order):
+  * If no Driver was assigned yet, status will be `cancelled`, meaning you won't be charged.
+  * If a Driver was already assigned, status will be `cancelledWithCharge`, meaning we will charge you the minimun value for a delivery.
```
Comment (gabisurita) [inline src/docs/presto/order-lifecycle/index.md]: typo: minimum

File: src/docs/presto/overview/index.md [hunk 10]
```diff
@@ -15,3 +15,18 @@ With the ```shopId``` in hands, you can follow our recommended work-flow:
...
+* Shop: indicates the pickup location
+* Package: indicates the delivery location
+* Order: a collection of packages - a single Driver is assigned to perform the delivery
+
+#### Shop
+* You can’t create a Shop through our API, this is done through our support.
+* You can list all shops assigned to your account. You’ll need this endpoint to find the Shop ID, which is required when creating an Order.
+
+#### Package
+* A package is composed by three groups: Recipient, Destination & Charge Information
```
Comment (gabisurita) [inline src/docs/presto/overview/index.md]: groups/components

**Pull request merged**